src/components/Dialog/HotkeySettingDialog.vue

```diff
@@ -298,26 +335,98 @@ const changeHotkeySettings = (
   });
 };
 
-const duplicatedHotkey = computed(() => {
-  if (lastRecord.value == "") return undefined;
-  return hotkeySettings.value.find(
-    (item) =>
-      item.combination == lastRecord.value && item.action != lastAction.value
+const duplicatedHotkeys = computed(() => {
+  return filterDuplicatedHotkeysWithSetKeyAndName(
+    lastRecord.value,
+    lastAction.value as HotkeyActionNameType
   );
 });
 
+const filterDuplicatedHotkeysWithSetKeyAndName = (
+  key: HotkeyCombination,
+  action: HotkeyActionNameType
+): HotkeySettingType[] => {
+  if (key == "") return [];
+  if (lastArgumentKeyText.value == "") {
+    const result = findDuplicatedHotkeyWithKey(key, action);
+    if (result == undefined) {
+      return [];
+    } else {
+      return [result];
+    }
```

Comment:
こういうとこは三項演算子を使うとすっきりするかもです！
こう？
```suggestion
    return result  == undefined ? [] : [result];
```